Intersect.Server/Entities/Entity.cs

```diff
@@ -2438,9 +2419,29 @@ public virtual void Die(int dropitems = 0, Entity killer = null)
                         continue;
                     }
 
+                    // Decide if we want to have a loot ownership timer or not.
+                    Guid lootOwner = Guid.Empty;
+                    if (this is Npc)
```

Comment:
`this is Npc npc` and replace the below cast with just `npc` (or if `npc` is already taken use `thisNpc` so it's clear that it's still a reference to `this`).